src/ui/users/list/UserListMenuActions.js

```diff
@@ -116,14 +116,14 @@ UserListMenuActions.propTypes = {
   onClickView: PropTypes.func,
   username: PropTypes.string.isRequired,
   hasProfile: PropTypes.bool,
-  canUser: PropTypes.func,
+  userPermissions: PropTypes.arrayOf(PropTypes.string),
 };
 
 UserListMenuActions.defaultProps = {
   onClickDelete: () => {},
   onClickView: () => {},
   hasProfile: false,
-  canUser: () => true,
+  userPermissions: [],
 };
 
 export default UserListMenuActions;
```

Comment:
Its more a question here. Since you do not wrap it with HOC, how does `UserListMenuActions` have access to `userPermissions` ?